internal/backends/postgresql/query.go

```diff
@@ -63,9 +63,12 @@ func prepareSelectClause(schema, table string, capped, onlyRecordIDs bool) strin
 }
 
 // prepareWhereClause adds WHERE clause with given filters to the query and returns the query and arguments.
-func prepareWhereClause(p *metadata.Placeholder, sqlFilters *types.Document) (string, []any, error) {
+func prepareWhereClause(p *metadata.Placeholder, sqlFilters *types.Document) (string, []any, bool, error) {
```

Comment:
As we start to return too many values, I'd suggest to create respectively `whereClauseResults`, and `orderByClauseResults`.